hack/gen-contributions.sh

```diff
@@ -42,8 +42,9 @@ The Linux Foundation® (TLF) has registered trademarks and uses trademarks. For
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fpipe-cd%2Fpipecd.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2Fpipe-cd%2Fpipecd?ref=badge_large)
 EOF
 
-# Remove fossabot from the contributors list.
+# Remove bots from the contributors list.
 sed -i '/fossabot/d' README.md.tmp
+sed -i '/github-actions[bot]/d' README.md.tmp
```

Comment:
these symbols need to be escaped.

```suggestion
sed -i '/github-actions\[bot\]/d' README.md.tmp
```

Reply:
@Warashi Thanks, I fixed it here-> [4b7604f](https://github.com/pipe-cd/pipecd/pull/4993/commits/4b7604f283cc0113c2e9ab2ef7e85d8395efbcbc)